### DISCUSSION TITLE
M30 Bluetooth controller don't work d-pad

### DISCUSSION THREAD

**Discussion by Metro-City** [darthcloud/BlueRetro#1065] [Q&A]
Hi, I have the blueretro MD doing the Start+X synchronization and then press sync, all the buttons work except the d-pad, how can this be solved? thanks

**Comment by darthcloud** [comment]
https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#73---d-pad-as-joystick-or-d-pad-configuration
